docs/api/ethereum.md

```diff
@@ -1,5 +1,645 @@
+  console.log('Network has changed', networkVersion);
+});
+```
+
+**See also:**
+
+[on (event, handler)](#on-event-handler), [off (event, handler)](#off-event-handler)
+
+### off(event, handler)
+
+A `synchronous` class instance `function` which removes an existing event handler.
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| event | [required] A `string` representing a [provider event](./ethereum.md#provider-events) name.
+| handler | A specific callback `function` of an event. If not provided, all handlers of the `event` are removed.
+
+**Result:**
+
+An instance of the same provider  class.
+
+**Example:**
+
+```ts
+import { ProviderEvent } from '@0xcert/ethereum-bitski-backend-provider';
+
+provider.off(ProviderEvent.NETWORK_CHANGE);
+```
+
+**See also:**
+
+[on (event, handler)](#on-event-handler), [once (event, handler)](#once-event-handler)
+
+### orderGatewayId
+
+A class instance `variable` holding a `string` which represents an Ethereum address of the [order gateway](/#public-addresses).
+
+### requiredConfirmations
+
+A class instance `variable` holding a `string` which represents the number of confirmations needed for mutations to be considered confirmed. It defaults to `1`.
+
+### signMethod
+
+A class instance `variable` holding a `string` which holds a type of signature that will be used (e.g. when creating claims).
+
+### unsafeRecipientIds
+
+A class instance `variable` holding a `string` which represents smart contract addresses that do not support safe ERC-721 transfers.
+
+### valueLedgerSource
+
+A class instance `variable` holding a `string` which represents the URL to the compiled ERC-20 related smart contract definition file. This file is used when deploying new value ledgers to the network.
+
+
+## Bitski front-end provider
+
+A [Bitski](https://www.bitski.com/) front-end provider is applied for in-browser use. The user will be prompted to create/login into his bitski account when creating mutations. The provider automatically establishes a communication channel to Bitski which further performs communication with the Ethereum blockchain. To setup Bitski front-end provider it is required to first create a [development account](https://developer.bitski.com/) on Bitski.
+
+### BitskiProvider(options)
+
+A `class` providing the communication with the Ethereum blockchain through [Bitski](https://www.bitski.com/).
+
+**Arguments**
+
+| Argument | Description
+|-|-|-
+| options.assetLedgerSource | A `string` representing the URL to the compiled ERC-721 related smart contract definition file. This file is used when deploying new asset ledgers to the network.
+| options.clientId | A `string` representing the Bitski client id. You get the client ID by creating a [developer account](https://developer.bitski.com/) on Bitski.
+| options.mutationTimeout | A `number` representing the number of milliseconds in which a mutation times out. Defaults to `3600000`. You can set it to `-1` for disable timeout.
```

Comment:
`| options.mutationTimeout | A `number` representing the number of milliseconds in which a mutation times out. Defaults to `3600000`. You can set it to `-1` to disable timeout.`